src/modules/variable/mod.rs

```diff
@@ -17,32 +17,15 @@ pub fn variable_name_extensions() -> Vec<char> {
 
 pub fn variable_name_keywords() -> Vec<&'static str> {
     vec![
-        // Literals
-        "true", "false", "null",
-        // Variable keywords
-        "let", "as", "is", "const",
-        // Control flow keywords
-        "if", "then", "else",
-        // Loop keywords
-        "for", "loop", "break", "continue", "in", "while",
-        // Module keywords
-        "pub", "import", "from",
-        // Function keywords
-        "fun", "return", "ref", "fail", "failed", "succeeded", "then",
-        // Types
-        "Text", "Number", "Bool", "Null",
-        // Command Modifiers
-        "silent", "trust", "sudo",
-        // Misc
-        "echo", "status", "nameof", "mv", "cd",
-        "exit", "len",
+        "loop", "main", "mv", "nameof", "not", "or", "pub", "ref", "return", "silent", "status",
+        "sudo", "succeeded", "then", "trust", "exited", "unsafe", "while",
```

Comment:
Why some of them are removed?

Reply:
I used `keywords.ab` to get all keywords in Amber and replaced the list with the new one. I just realized that not only keywords are invalid identifiers but also literals.